astro/src/content/docs/get-started/download-and-install/system-requirements.mdx

```diff
@@ -30,7 +30,13 @@ Docker, k8s and other container platforms are supported because the host operati
 
 FusionAuth ships with its own version of Java. The version that is contained in the bundle is the required and supported version.
 
-Currently, Java 17 is the supported major version. Java 14 is the supported major version for all FusionAuth releases before FusionAuth 1.32.
+Currently, Java 21 is the supported major version. The versions of Java that are supported for prior versions of FusionAuth are:
```

Comment:
Looks like our runner / vale is broken? 
We of course don't want to use `fusionauth` instead of `FusionAuth`. 

I had a similar issue in my PR for using `GitHub` and it suggested I change it. 🤷 
- https://github.com/FusionAuth/fusionauth-site/pull/3248/files#diff-a0f96c3147ae37620b622e94c93f5b98944844b6e92bb0de1b664b5dc7212cb6